### DISCUSSION TITLE
Please explain "Inpaint at full resolution"

### DISCUSSION THREAD

**Discussion by Polukeev** [zero01101/openOutpaint#265] [Q&A] [answered]
Hello world, hello @zero01101 and thank you for making the best inpainting extension for automatic's sd-web-ui.
I have a question about "Inpaint at full resolution" checkbox feature when you use img2img. I like to use it with small de-noising strength to make objects more detailed, but i really don't understand what does this feature do? I do get how it works in sd, but here is little bit different since canvas is dynamically change during outpainting. I didn't find the answer by googling, so i have decided to ask a creator of the software product :)
![123](https://github.com/zero01101/openOutpaint/assets/45034807/6697abdb-bec9-48d7-824d-f328b3d9df46)
Do i understand this feature correctly? With this setting applied at the object is starting to look more detailed even more detailed than overall picture, when i uncheck it the object i want to inpaint becomes more blurry but it fits to overall composition.

**Comment by zero01101** [comment, marked as answer]
ha ha ha 
![lil bad pokerface](https://github.com/zero01101/openOutpaint/assets/1649724/3e66f95d-e3fb-41b9-84a8-9cec4fdc2b42)
i honestly just forget that option exists and just leave it enabled myself, but you've got the right idea - the intent is to scale the masked region to the size of the entire image, do the inpaint maneuvers, and scale it back down as per https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#inpaint-area - thanks for the kind words and my apologies for missing this for so long :)
